analyze/index.ts

```diff
@@ -1,22 +1,17 @@
-import type { ArcjetLogger } from "@arcjet/protocol";
-
-import { instantiate } from "./wasm/arcjet_analyze_js_req.component.js";
+import { initializeWasm } from "@arcjet/analyze-wasm";
 import type {
-  ImportObject,
+  AnalyzeContext,
+  BotConfig,
+  BotResult,
+  DetectedSensitiveInfoEntity,
+  DetectSensitiveInfoFunction,
   EmailValidationConfig,
   EmailValidationResult,
-  DetectedSensitiveInfoEntity,
   SensitiveInfoEntities,
   SensitiveInfoEntity,
   SensitiveInfoResult,
-  BotConfig,
-  BotResult,
-} from "./wasm/arcjet_analyze_js_req.component.js";
-import type { ArcjetJsReqSensitiveInformationIdentifier } from "./wasm/interfaces/arcjet-js-req-sensitive-information-identifier.js";
-
-import { wasm as componentCoreWasm } from "./wasm/arcjet_analyze_js_req.component.core.wasm?js";
-import { wasm as componentCore2Wasm } from "./wasm/arcjet_analyze_js_req.component.core2.wasm?js";
-import { wasm as componentCore3Wasm } from "./wasm/arcjet_analyze_js_req.component.core3.wasm?js";
+} from "@arcjet/analyze-wasm";
+import type { ImportObject } from "@arcjet/analyze-wasm/wasm/arcjet_analyze_js_req.component";
```

Comment:
```suggestion
```
I thought you re-exported this? We shouldn't reach into a subdirectory to load the type.